### ISSUE TITLE
Concerns related to Art 7

### ISSUE THREAD

**Issue by dkoureas** [tdwg/constitution#4]
The following was received by a TDWG member:

> As it is currently drafted there is a huge importance on the controls and scope of actions/responsibilities built into the FS Charter – this makes wording and approval of the Charter extremely important.  
> I strongly believe that we need to update Article 7 to allow for provision of change to a FS in both Charter and membership, and to retain the ability of some Executive influence.  
> 
> In particular Article 7
> •	Does not include provision for modification of a FS Charter.   
> Suggest a sentence needs to be added to this effect.  It would seem appropriate that modifications  may happen at any time at request of the Executive Committee or be initiated by the Chairperson of the FS. Once drafted this would probably trigger the consultation with membership and approval of modified Charter by Exec.
> •	Article 7.2 –  do NOT agree with FS automatically operating for remainder of Chairs term after it has been dissolved by the Executive.
> I believe this needs to be amended to be “Following the Executive Committee’s decision to dissolve a FS, the Executive Committee may choose to dissolve the FS with immediate effect or at the end of the FS chairperson’s term of duty.”
> •	Article 7.3 – I believe that  the Executive should also have the power to nominate (and possibly veto) members for a FS.

**Comment by peterdesmet** [comment]
The requested updates seem reasonable, especially 7.1 and 7.2

**Comment by dkoureas** [comment]
> I strongly believe that we need to update Article 7 to allow for provision of change to a FS in both Charter and membership, and to retain the ability of some Executive influence.

The Executive Committee is responsible for setting the thematic priorities for the creation and operation of the Functional Subcommittees. In that context it retains the capacity to change these priorities as needed. However, the Functional Subcommittees do receive membership mandate for their operation (direct election of chairs and vice-chairs). 

> • Does not include provision for modification of a FS Charter.
> Suggest a sentence needs to be added to this effect. It would seem appropriate that modifications may happen at any time at request of the Executive Committee or be initiated by the Chairperson of the FS. Once drafted this would probably trigger the consultation with membership and approval of modified Charter by Exec.

Art 7.2 sets the process for the approval of charters by the Executive Committee, and does not exclude the process of updating the charters. These provisions will need to be made at statute level.

> • Article 7.2 – do NOT agree with FS automatically operating for remainder of Chairs term after it has been dissolved by the Executive.
> I believe this needs to be amended to be “Following the Executive Committee’s decision to dissolve a FS, the Executive Committee may choose to dissolve the FS with immediate effect or at the end of the FS chairperson’s term of duty.”

As mentioned above FS chairs have direct membership mandate and participate as full members of the Executive Committee. As such their term cannot be terminated by the power of the Executive Committee. 

> • Article 7.3 – I believe that the Executive should also have the power to nominate (and possibly veto) members for a FS.

Nominations of individuals to the Chairs of the FS is not excluded. XC can nominate people at will. Again, I don't see a reason to include this in the Constitution. The FS chairs are full members of the XC and these discussions can be handled as part of the remit of the XC.

**Comment by jmacklin** [comment]
I generally agree with Dimitris. For 7.2, the XC can dissolve an FS and in doing so, the Chair would default as a general member of the XC. In this case, we would need to be clear somehow that when the membership votes for a given Chair and thus XC member that this is not exclusive. Do we have a mechanism to remove an XC member if necessary? I don't think it is wise to do this based on performance but it would be appropriate for some kind of acts obviously (criminal, discriminatory...). Sorry did not take the time to check ;-)

JAmes

**Comment by ghwhitbread** [comment]
Also in general agreement with Dimitris ... though changes to FS charter (including dissolution) would seem to require 21 days open consultation with the membership.  Probably a good thing, but there will be an impact on the nature of charters in that they will be written in very general terms (and TAG draft charter will change) with the actual workings of a committee documented elsewhere.

But I could not figure out what to answer to the survey monkey questions

**Comment by dkoureas** [comment]
@jmacklin Art. 3 of the current (and revised) Constitution specifically mentions end of membership conditions. Professional misconduct is one of them. All XC members are subject to this Article as they need to be TDWG members to participate to the TDWG XC.

**Comment by jmacklin** [comment]
Thanks Dimitris for clarification. So, this makes me believe that there
does not need to be any revision of the current charter as the XC does have
process to manage and dissolve FSs and deal with the person/people
associated.

James

**Comment by stanblum** [comment]
+1 for keeping the language as it is, just to keep things simple.

I think we should keep the option open, pending further discussion, to propose another small amendment to alter article 7, but only after this larger proposal passes.

**Comment by dkoureas** [comment]
Hi @stanblum Stan,

Are you planning to circulate the ballots invites today?

**Comment by stanblum** [comment]
Yes,  It will give the links to the package and page on the web site, and
say something like:

I am an [ individual / institutional ] member of TDWG and I:

approve the constitutional proposal

do NOT approve the constitutional proposal

Two separate surveys will be used, one for individuals and the other for
institutions.  All individuals/institutions who paid membership (or were
granted membership as being from a developing country) in 2015 or 2016 will
get a ballot.

Any other issues I should address?

-Stan

**Comment by dkoureas** [comment]
To be on the safe side, I would propose to also include a URL (github?) that points to the Constitution proposal package in the ballot itself.

Kind regards,
Dimitris

## #End

Dr Dimitrios Koureas, FLS
Lead, Research Data & Partnerships
Dept Life Sciences | Natural History Museum London | SW7 5BD, UK

Coordinator, Distributed System of Scientific Collections (DiSSCo) initiative
Executive Secretary, Biodiversity Information Standards (TDWG)
co-Chair, RDA – Biodiversity Data Integration IG & Attribution metadata WG

ORCID: 0000-0002-4842-6487 | Linkedin: uk.linkedin.com/in/dkoureas
E-mail: d.koureas@nhm.ac.uk | Twitter: @DimitrisKoureas | Tel: +44 (0) 207 942 5244

**Comment by stanblum** [comment]
OK, I will also add an explanation on the repo homepage that there are two
branches, current and proposed, and to view the two versions, the branch
dropdown can be used to switch between them.

**Comment by peterdesmet** [comment]
Here's a link to the proposal:
https://github.com/tdwg/constitution/blob/49a9c6c5f86f93e36d16f4958b772d98668b6864/constitution.md

And here's a link to see the changes:
https://github.com/tdwg/constitution/pull/1/files?short_path=26f07ab#diff-26f07ab58c6acb6da1a3d79f4c13125d
